CHANGELOG.md

```diff
@@ -2,7 +2,7 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-_(none)_
+* Update to v2.18.0 of the NewRelic Terraform Provider
```

Comment:
```suggestion
* Update to v2.19.0 of the NewRelic Terraform Provider
```

Reply:
Good catch! Will fix as part of the release